doc/sphinx/changes.rst

```diff
@@ -11,9 +11,184 @@ Recent changes
+and Enrico Tassi with contributions from many users. The up-to-data list
+of packages is `available on the Rocq website <https://rocq-prover.org/packages>`_.
+
+Rocq 9.0 was made possible thanks to the following reviewers:
+TODO
+
+See the `Rocq Team <https://rocq-prover.org/governance>`_ page for
+more details on Rocq's development teams.
+
+The ?? contributors to the 9.0 version are:
+TODO
+
+The Coq/Rocq community at large helped improve this new version via
+the GitHub issue and pull request system, the coq-club@inria.fr mailing list,
+the `Discourse forum <https://coq.discourse.group/>`_ and the
+`Coq Zulip chat <https://coq.zulipchat.com>`_.
+
+Version 9.0's development spanned 7 months from the release of Coq 8.20.0.
+Pierre-Marie Pédrot and Matthieu Sozeau are the release managers of Rocq 9.0.
+This release is the result of ?? merged PRs, closing ?? issues.
+
+| Nantes, January 2025
+| Pierre-Marie Pédrot and Matthieu Sozeau for the Rocq development team
+
+Porting to The Rocq Prover
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Rocq Prover version 9.0 includes support for compability shims that allow
+to call it through legacy Coq commands (`coq-tex`, `coq_makefile`, `coqchk`, `coqdoc`, `coqpp`, `coqtop`,
+`coqwc`, `coqc`, `coqdep`, `coqnative`, `coqtimelog2html`, `coqtop.byte`, `coqworkmgr`)
+so that it can act as a regular `Coq` version. If using `opam`, this compatibility
+layer is available through packages `coq-core`, `coq-stdlib` and `coq`. In this setting,
+nothing needs to be changed to an existing project to compile with `Rocq 9.0` (aliased as `Coq 9.0`).
+
+You should expect warnings that the standard library previously under namespace
+`Coq` has been renamed to `Stdlib` (and moved to a `separate repository
+<https://github.com/coq/stdlib>`_), but these warnings are independent of the
+`Coq` to `Rocq` renaming. See `this entry
+<https://rocq-prover.org/doc/v9.0/refman-stdlib/changes.html#changed>`_ from
+the Standard Library's changelog for the suggested workflow to port theories.
+
+There are important changes to consider for building plugins and libraries:
+
+- To be future-proof, projects based on `coq_makefile` can be ported to not rely
+  on the compatibility layer anymore. To do so, one must replace uses of
+  `coq_makefile` with :ref:`rocq makefile <rocq_makefile>`, which will directly
+  call the new `rocq` binary without relying on the compatibility shims.
+
+- If using `dune` (TODO <= 3.17.2) to :ref:`build <building_dune>` a Rocq project, you will still
+  need the compatibility shim for `coq-core` so that `dune`'s Coq language extension functions correctly.
+  This issue has been solved in `dune` TODO.
+
+Regarding packaging:
+
+- Opam packages depending on the coq shim should keep being named coq-\*, whereas
+  actually ported packages should be named rocq-\* and replace the coq dependency by
+  `rocq-core` or `rocq-stdlib` (but **not** `rocq-prover` which is only a user-oriented
```

Comment:
```suggestion
  `rocq-core` and `rocq-stdlib` (but **not** `rocq-prover` which is only a user-oriented
```

Reply:
No, it's really an or. If only because you may not depend on rocq-stdlib.

Reply:
(of course, one may want to put version bounds on both rocq-core and rocq-stdlib, but that's another story)

Reply:
Let us not make it more complicated than needed. As it stands, the sentence looks like a user could replace the dependency on `coq` by a dependency on only `rocq-stdlib`. This is bad advice.

Reply:
Hmm, I added a parenthetical remark about packages not depending on rocq-stdlib. Does it satisfy everyone?

Reply:
LGTM, thanks!